src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import './TodoInfo.scss';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    key={todo.id}
```

Comment:
The 'key' attribute should not be used here. It's generally used in the context of an array map function.